programming/sed-article.md

```diff
@@ -417,14 +401,14 @@ and placing a `?` after the group.
 
 ## Two `sed` Buffers
 
-Text read by `sed` is temporarily stored in two buffers
-referred to as "PatSpace" and "HoldSpace".
```

Comment:
Have you not seen the terms "PatSpace" and "HoldSpace" used in descriptions of sed?
I got those from the book I read on sed.

Reply:
Most of my books spell out pattern space and hold space.  I would go with what's in the manpage.

Reply:
Done.